libs/api/domains/driving-license/src/lib/drivingLicense.service.ts

```diff
@@ -52,10 +52,12 @@ export class DrivingLicenseService {
 
   async getDrivingLicense(
     nationalId: User['nationalId'],
+    token?: string,
```

Comment:
For now, this is optional and is used in v5 fallbacks when a disqualification is present on the driving license.